src/components/Trigger/Trigger.tsx

```diff
@@ -56,7 +60,7 @@ export const Trigger: FC<TriggerProps> = ({
             {children}
             <div
                 className={merge([
-                    "tw-flex-none tw-flex tw-items-center tw-absolute",
+                    "tw-flex-none tw-flex tw-items-center tw-absolute tw-gap-2",
                     size === TriggerSize.Large ? "tw-right-5 tw-gap-1.5" : "tw-right-3 tw-gap-1",
```

Comment:
The gap is already set according to the TriggerSize, so I'd remove `tw-gap-2` again and leave the gap size as is

Reply:
Changed